app/views/asset/WhatKindOfAssetView.scala.html

```diff
@@ -34,23 +38,23 @@
 }
 
 @main_template(
-    title = s"${errorPrefix(form)} $title"
+    title = s"${errorPrefix(form)} $title",
+    showBackLink = true
     ) {
 
     @formHelper(action = WhatKindOfAssetController.onSubmit(index, draftId), 'autoComplete -> "off") {
 
-        @components.back_link()
+        @errorSummary(form.errors)
 
-        @components.error_summary(form.errors)
-
-        @components.input_radio(
+        @inputRadio(
             field = form("value"),
             legend = heading.toString,
-            legendClass = Some("heading-large"),
-            inputs = options,
-            legendAsHeading = true
+            legendClass = Some("govuk-heading-l"),
+            inputs = mapRadioOptionsToRadioItems(form("value"), trackGa = false, options),
```

Comment:
The options need to be fed into the error summary, for the error summary link properly focus on the first input. 

`@error_summary(form.errors, options)`

See commit here https://github.com/hmrc/register-trust-settlor-frontend/pull/70/commits/6dde61a9f54ba5a0960f30766c2a598b974478bb

I also think options needs to be type Seq - for the mapRadioOptionsToRadioItems method in ViewUtils

Reply:
Done